paasta_tools/api/api_docs/swagger.json

```diff
@@ -1293,8 +1293,38 @@
                     "description": "The status of the pod"
                 },
                 "tail_lines": {
-                    "$ref": "#/definitions/TaskTailLines",
-                    "description": "Stdout and stderr tail of the task"
+                    "$ref": "#/definitions/PodTailLines",
+                    "description": "Stdout and stderr tail for each container in the pod"
+                }
+            }
+        },
+        "PodTailLines": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "container": {
```

Comment:
Could we make "container" as the properties for kubernetesPod? By doing that, we don't need to define PodTailLines.

Reply:
I've added a containers array to the `KubernetesPod` object in the swagger spec, which includes the `tail_lines`. I agree it's more intuitive.